internal/timecraft/module.go

```diff
@@ -15,6 +15,10 @@ type ModuleSpec struct {
 	// Path is the path of the WebAssembly module.
 	Path string
 
+	// Name of the exported function to call in the WebAssembly module.
+	// If empty, the _start function will be executed.
+	Function string
```

Comment:
At first sight it seems the use cases for executing anything other than _start (which has no args and no return value) might be very limited, do we need a way to pass arguments and retrieve the results?

Reply:
Did you look at how extism does it? It seems they've got a somewhat cross-language mechanism for invoking exported functions with rich parameters

Reply:
They use a C binding. Similar to wit-binding or spin.

Reply:
But how do they solve for the calling convention? The data needs to be passed between the host and the guest, there's more than C bindings to achieve that.

Reply:
Will be part of another PR.